src/WebpubManifestTypes/ConformsTo.ts

```diff
@@ -2,7 +2,11 @@
 export const WebpubPdfConformsTo = 'stub/webpub+pdf';
 // a webpub of axisnow content
 export const AxisNowEpubConformsTo = 'stub/webpub+axisnow';
+// Epub profiles
+export const EpubConformsTo =
+  'https://readium.org/webpub-manifest/profiles/epub';
```

Comment:
This is what the Readium team uses. I am curious how did we come up with 'stub/webpub+pdf'?

Reply:
Well we were meant to replace that with a profile we defined and published somewhere, but I'm not sure that we ever actually did publish a profile somewhere. 

Separately, this does pose a problem... we need AxisNow books to conform to the AxisNowEpubConformsTo... I believe we use this in web-reader to know that it is encrypted with axisnow. However, it may be that we should be using the [encryption module](https://github.com/readium/webpub-manifest/blob/master/modules/encryption.md) for that purpose. So we can declare that this conforms to the Epub spec with AxisNow encryption

Reply:
Actually we do already declare that the individual resources are encrypted with the AxisNow scheme. So we can remove the `AxisNowEpubConformsTo` I believe